docs/conf.py

```diff
@@ -4,9 +4,15 @@
 # list see the documentation:
 # https://www.sphinx-doc.org/en/master/usage/configuration.html
 
+import subprocess
 from rocm_docs import ROCmDocs
 
-docs_core = ROCmDocs("hipBLAS Documentation")
+getVersionCmd = r'sed -n -e "s/^.*VERSION_STRING.* \"\([0-9\.]\{1,\}\).*/\1/p" ../CMakeLists.txt'
```

Comment:
this should be in rocm-docs-core. But please test here first if you want

Reply:
reverted. 
44521d6